indico/modules/categories/controllers/display.py

```diff
@@ -104,7 +104,7 @@ def _process(self):
             'total_events': sum(stats['events_by_year'].values()),
             'total_contributions': sum(stats['contribs_by_year'].values()),
             'min_year': stats['min_year'].year,
-            'max_year': date.today().year,
+            'max_year': stats['max_year'].year,
```

Comment:
The point of using the current year here was to avoid getting useless data when you have people adding events that end a few decades from now. That's why the JS logic uses up to `maxYear + 3` (if there's data for that long) when displaying the chart.

Reply:
I see.. I was actually wondering why it has been implemented like that. In this case, shouldn't we limit the categories/contributions only to those happening before `date.today().year`?